backend/cmd/migrate/migrations/000004_experiments_schema_changes.up.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS experiments;
+CREATE TABLE IF NOT EXISTS experiment_config (
+    id BIGINT,
+    details JSONB
```

Comment:
Does not have to be a part of this PR.

One thing that we haven't touched on yet - do you think that there would be any value in us adding some kind of a `string type` field in here to help packages depending on `experimentation` package to query the tests they are interested in? 

As of now, the only way to learn about the type of a given experiment config is to look at the value stored for `@type` in `details` JSON (i.e. if the value is equal to `type.googleapis.com/clutch.chaos.serverexperimentation.v1.TestConfig` then we are dealing with a server side fault injection test). //cc @donyu since he will need to query these tables when he started pushing postgres changes to other parts of Lyft infra.

Reply:
I think adding a type might make sense. You can currently determine the type during queries by using the postgres json syntax but it's cumbersome.

Reply:
Do you think we could at the column itself as part of this PR to avoid SQL migration going forward or would it be better to add the column in the follow up PR?

Reply:
Yes we'll need some easy way to filter out only mobile fault injection scenarios.

Reply:
https://jira.lyft.net/browse/RES-1369